packages/react/package.json

```diff
@@ -49,9 +49,9 @@
   "dependencies": {
     "@babel/runtime": "^7.18.3",
     "@carbon/feature-flags": "^0.19.0",
-    "@carbon/icons-react": "^11.40.0-rc.0",
+    "@carbon/icons-react": "^11.40.0-rc.1",
     "@carbon/layout": "^11.21.0",
-    "@carbon/styles": "^1.55.0",
+    "@carbon/styles": "^1.55.1-rc.0",
```

Comment:
`@carbon/styles` version

Reply:
I think so, the last release had no changes to styles. https://github.com/carbon-design-system/carbon/pull/16185/files